src/infra/core/src/ingest_service_impl.rs

```diff
@@ -110,17 +109,34 @@ impl IngestServiceImpl {
         let listener =
             get_listener(&dataset_handle).unwrap_or_else(|| Arc::new(NullIngestListener));
 
+        let some_random_hash: Multihash =
+            Multihash::from_multibase_str("zW1a3CNT52HXiJNniLkWMeev3CPRy9QiNRMWGyTrVNg4hY8")
+                .unwrap();
```

Comment:
Consider replacing with `Multihash::from_digest_sha3_256(b"")` not to hard code a weird string and make it more clear that this is not a hash of any real object

Reply:
Replaced